docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,234 @@
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic [working knowledge of Kubernetes](/docs/getting-started-guides/) in general, and [Services](/docs/user-guide/services/) in particular. 
+
+## Overview
+
+Federated Services are created in much that same way as traditional [Kubernetes Services](/docs/user-guide/services/)
+by making an API call which specifies the desired properties of your service. In the case of Federated Services, this API call is
+directed to the Federation API endpoint, rather than a Kubernetes cluster API endpoint.  The API for Federated Services is
+100% compatible with the API for traditional Kubernetes Services.
+
+Once created, the Federated Service automatically:
+
+1. creates matching Kubernetes Services in every cluster underlying your Cluster Federation,
+2. monitors the health of those service "shards" (and the clusters in which they reside), and
+3. manages a set of DNS records in a public DNS provder (like Google Cloud DNS, or AWS Route 53), thus ensuring that clients
+of your federated service can seamlessly locate an appropriate healthy service endpoint at all times, even in the event of cluster,
+availability zone or regional outages.
+
+Clients inside your federated Kubernetes clusters (i.e. Pods) will automatically find the local shard of the Federated Service in their cluster
+if it exists and is healthy, or the closest healthy shard in a different cluster if it does not.
+
+## Hybrid Cloud Capabilities
+
+Federations of Kubernetes Clusters can include clusters running in different cloud providers (e.g. Google Cloud, AWS), and on-premise (e.g. on OpenStack).  Simply create all of the clusters that you require, in the appropriate cloud providers and/or locations, and register each cluster's API endpoint and credentials with your Federation API Server  (See the [federation admin guide](/docs/admin/federation/) for details).
+
+Thereafter, your applications and services can span different clusters and clout providers as described in more detail below.
+
+## Creating a Federated Service
+
+This is done in the usual way, for example:
+
+```
+kubectl --context=federation-cluster create -f services/nginx.yaml
+```
+
+The '--context=federation-cluster' flag tells kubectl to submit the request to the Federation API endpoint, with the appropriate credentials.
+If you have not yet configured such a context, visit the  [federation admin guide](/docs/admin/federation/) or one of the
+[administration tutorials](https://github.com/kelseyhightower/kubernetes-cluster-federation) to find out how to do so.
+
+As described above, the Federated Service will automatically create and maintain matching Kubernetes services in all of the clusters underlying your federation.
+
+You can verify this by checking in each of the underlying clusters, for example:
+
+```
+kubectl --context=gce-asia-east1-a get services nginx
```

Comment:
I think they work fine with no namespace, or the namespace configured in the context. I'd rather not clutter the example with the --namespace argument given that it's largely irrelevant?  OK with that?